lib/resourceapply/machineconfig_test.go

```diff
@@ -4,7 +4,7 @@ import (
 	"fmt"
 	"testing"
 
-	ignv2_2types "github.com/coreos/ignition/config/v2_2/types"
+	ignvtypes "github.com/coreos/ignition/config/v3_0/types"
```

Comment:
love this :100:

Reply:
why ignvtypes and not igntypes?

Reply:
ha yeah i caught that one!!  thank you for :eyes: @LorbusChris